libraries/engage/core/actions/requestAppPermission.js

```diff
@@ -0,0 +1,26 @@
+import { requestAppPermissions } from '@shopgate/pwa-core/commands/appPermissions';
+import { PERMISSION_STATUS_NOT_SUPPORTED } from '@shopgate/engage/core/constants';
+import { appPermissionStatusReceived } from '../action-creators';
+
+/**
+ * Requests an additional app permission from the operating system.
+ * Additionally it propagates * the received status via the APP_PERMISSION_STATUS_RECEIVED action.
+ * @param {Object} params The action params
+ * @param {string} params.permissionId The desired app permission id
+ * @returns {Function} A Redux thunk
+ */
+const requestAppPermission = ({ permissionId }) => async (dispatch) => {
```

Comment:
For me, the two names "requestAppPermission" and "requestAppPermissionStatus'" are super confusing.
Why not keep the existing names "getAppPermission" and "requestAppPermission"?